### ISSUE TITLE
Currently, working? Invalid license key

### ISSUE THREAD

**Issue by blankster** [ozh/yourls-geoip-update#4]
Thanks for the plugin in general 👍 

I'm getting currently:

```
Database not updated.

Error downloading "GeoLite2-Country". Invalid license key.
```

I've tried https://download.maxmind.com/app/geoip_download?edition_id=GeoLite2-Country&license_key=YOUR_LICENSE_KEY&suffix=tar.gz to be sure, that my key is working, and I could download with the same key (and validated it multiple times) without problems. 

If it should work, how can I further debug this problem? Is it possible that the plugin is currently broken with the latest YOURLs 1.9.2?

What I've noticed is that my license key looks drastically longer (40 characters long) than the one on the screenshot in the README.md. Is it possible that something regarding the key format has changed?

**Comment by fribse** [comment]
That made it work for me, thankyou!

**Comment by ozh** [comment]
Fixed via #5
